webapp/components/post_view/components/post_list.jsx

```diff
@@ -236,15 +238,35 @@ export default class PostList extends React.Component {
             }
 
             let commentCount = 0;
+            let nonOwnCommentsExists = false;
+            let isCommentMention = false;
             let commentRootId;
             if (parentPost) {
                 commentRootId = post.root_id;
             } else {
                 commentRootId = post.id;
             }
-            for (const postId in posts) {
-                if (posts[postId].root_id === commentRootId) {
-                    commentCount += 1;
+            if (commentRootId) {
+                const commentsNotifyLevel = UserStore.getCurrentUser().notify_props.comments || 'never';
```

Comment:
Minor, but we already pass in the `currentUser` through props. Could you use that instead of going to the stores again? Don't forget to remove the import when you make this change :)